dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@split-part-negative-part-number#egg=dbt-core&subdirectory=core
```

Comment:
Is there a corresponding change in `dbt-core` related to this? If not, you should be able to leave this as is. And either way, make sure to change this back before merging.

Reply:
Original change in `dbt-core` has been merged now:
https://github.com/dbt-labs/dbt-core/pull/7200

Will change this back momentarily.